design-editor/src/utils/path-utils.js

```diff
@@ -60,6 +61,53 @@ export default {
 	 */
 	getFileName(pathName, ext) {
 		return ext ? path.basename(pathName) : path.basename(pathName, path.extname(pathName));
+	},
+
+	/**
+	 * Set a source to background image or image
+	 * @param {Event} event
+	 * @param {string} changedAttribute name of changed attribute
+	 * @param {string} elementId id of changed element
+	 * @param {function} changeImageInfo function that changed info about element in sidebar
+	 */
+	setImageSource(event, changedAttribute, elementId, changeImageInfo) {
+		[].slice.call(event.target.files).forEach(file => {
```

Comment:
we should probably move it to another utils file (eg attribute-utils.js) as it does not exactly fit path-utils.